portals/new_admin/portals/client/src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title><%= htmlWebpackPlugin.options.title %></title>
+    <script type="text/javascript" src="/Settings.js"></script>
+  </head>
+  <body>
+    <div id="root"></div>
+  </body>
+</html>
```

Comment:
Shall we add an EOF new line here as well?

Reply:
Currently, there is no such file.